src/c++/uds/src/uds/numeric.h

```diff
@@ -6,7 +6,24 @@
 #ifndef UDS_NUMERIC_H
 #define UDS_NUMERIC_H
 
+#ifndef VDO_UPSTREAM
+#include <linux/version.h>
+#undef VDO_USE_ALTERNATE
+#if defined(RHEL_RELEASE_CODE) && defined(RHEL_MINOR) && (RHEL_MINOR < 50)
+#if (RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(9, 0))
```

Comment:
The RHEL_RELEASE_VERSION checks aren't consistent throughout your patch. There are checks for > 9.0 and for > 10.0. Is this correct?

Reply:
As Ken says, whatever checks we settle on should be used consistently in all the places you're touching.